javascripts/tagmanager.js

```diff
@@ -1673,8 +1673,26 @@
                 }
 
                 if (container.tags && utils.isArray(container.tags)) {
```

Comment:
@snake14 Can you also add a comment what are we doing here, its a bit hard to understand

Reply:
I didn't add line 1675, but I added some more inline comments in my new lines below.